Pinniped installed from `brew install pinniped-cli --HEAD` panics on `pinniped version`

To reproduce:
- `brew tap vmware-tanzu/pinniped`
- `brew install pinniped-cli --HEAD`
- `pinniped version`

### Error
```
panic: version string "?.?.?" doesn't match expected regular expression: "^v(\d+\.\d+\.\d+)"

goroutine 1 [running]:
k8s.io/component-base/metrics.parseVersion(0x28c0bc0, 0x1, 0x28c0bc2, 0x1, 0x28c0c2c, 0x5, 0x2906020, 0x28, 0x28c0c24, 0x5, ...)
	/Users/moyerm/Library/Caches/Homebrew/go_mod_cache/pkg/mod/k8s.io/component-base@v0.0.0-20210329195309-e1576f54c4ca/metrics/version_parser.go:47 +0x2c5
k8s.io/component-base/metrics.newKubeRegistry(0x28c0bc0, 0x1, 0x28c0bc2, 0x1, 0x28c0c2c, 0x5, 0x2906020, 0x28, 0x28c0c24, 0x5, ...)
	/Users/moyerm/Library/Caches/Homebrew/go_mod_cache/pkg/mod/k8s.io/component-base@v0.0.0-20210329195309-e1576f54c4ca/metrics/registry.go:312 +0xfb
k8s.io/component-base/metrics.NewKubeRegistry(0x9, 0xc0006b0280)
	/Users/moyerm/Library/Caches/Homebrew/go_mod_cache/pkg/mod/k8s.io/component-base@v0.0.0-20210329195309-e1576f54c4ca/metrics/registry.go:327 +0x7f
k8s.io/component-base/metrics/legacyregistry.init()
	/Users/moyerm/Library/Caches/Homebrew/go_mod_cache/pkg/mod/k8s.io/component-base@v0.0.0-20210329195309-e1576f54c4ca/metrics/legacyregistry/registry.go:29 +0x26
```

--- a/hack/lib/version.sh
+++ b/hack/lib/version.sh
@@ -55,9 +55,11 @@ kube::version::get_version_vars() {
 
     else
       # KUBE_GIT_VERSION was not supplied
-      KUBE_GIT_VERSION='?.?.?'
-      KUBE_GIT_MAJOR='?'
-      KUBE_GIT_MINOR='?'
+      # These values need to pass the validation in k8s.io/component-base/metrics:
+      # https://github.com/kubernetes/component-base/blob/v0.20.5/metrics/version_parser.go#L28-L50
+      KUBE_GIT_VERSION='0.0.0'
+      KUBE_GIT_MAJOR='0'
+      KUBE_GIT_MINOR='0'
     fi
   fi
 }

